src/components/ToyList.js

```diff
@@ -1,34 +1,43 @@
-import React, { useState } from "react";
-import { Link } from "react-router-dom";
-import { Container, Card, Row, Col, Button } from "react-bootstrap";
-import { sortToys, filterReview } from "../helpers/toyHelpers";
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Container, Card, Row, Col, Button } from 'react-bootstrap';
+import { sortToys, filterReview, searchToys } from '../helpers/toyHelpers';
+import Search from './SearchToys';
 
 const ToyList = ({ toys }) => {
-  const [sortDirection, setSortDirection] = useState("forward");
-  const [reviewFilter, setFilter] = useState("off");
+  const [sortDirection, setSortDirection] = useState('forward');
+  const [reviewFilter, setFilter] = useState('off');
+  const [searchInput, setSearchInput] = useState('');
 
   function HandleSortOptionChange() {
-    setSortDirection("forward");
+    setSortDirection('forward');
   }
 
   function HandleSortOptionChangeBackward() {
-    setSortDirection("backward");
+    setSortDirection('backward');
   }
 
   function HandleFilterWithReviews() {
-    setFilter("on");
+    setFilter('on');
+  }
+
+  function getSearchInput(input) {
+    setSearchInput(input);
```

Comment:
Some code smells here, calling a method named `get` that all it does it call a method named `set`?  Are you getting or setting, and why do you have a method that all it does it calls another method?